<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputFrame</class>
 <widget class="QFrame" name="InputFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>251</width>
    <height>135</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>230</width>
     <height>114</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="imageDirButton">
        <property name="text">
         <string>Image Dir</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="imageDirLineEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="outputDirButton">
        <property name="text">
         <string>Output Dir</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="outputDirLineEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="circlesFileButton">
        <property name="text">
         <string>Input Circles File</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="circlesFileLineEdit"/>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="scanFilesDirctoryButton">
      <property name="text">
       <string>Scan Files Directory</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
